config/default.go

```diff
@@ -104,7 +104,7 @@ SyncChunkSize = 100
 TrustedSequencerURL = "" # If it is empty or not specified, then the value is read from the smc
 SyncBlockProtection = "safe" # latest, finalized, safe
 L1SynchronizationMode = "sequential"
-L1SyncCheckL2BlockHash = true
+L1SyncCheckL2BlockHash = false
```

Comment:
What is the reason of disabling the check?

Reply:
to disable disable by default the problematic and not beneficial check that I suggested to remove in this other PR https://github.com/0xPolygonHermez/zkevm-node/pull/3575